perfd.py

```diff
@@ -24,14 +28,85 @@
         samples_per_obj_file[name] += 1
     else:
         samples_per_obj_file[name] = 1
+'''
 
+samples_per_obj_file = dict() # i will have to delete it when i finish the step 5
 database = db.Database()
 database.table = 'database'
 for name in samples_per_obj_file:
     database.write(
         perf_name=temp_perf_file.name,
         name_obj_file=name,
-        amount_of_samples=samples_per_obj_file[name])
+        number_of_samples=samples_per_obj_file[name])
```

Comment:
Имя аргумента `database.write` должно совпадать с именем колонки в db.py. Предлагаю тут оставить как было:
```suggestion
        amount_of_samples=samples_per_obj_file[name])
```